website/assets/css/image-gallery.css

```
@@ -0,0 +1,75 @@
+.image-gallery {
+    overflow: hidden;
+    padding-inline-start: 0;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "padding-inline-start" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)